testing/python/language/test_tilelang_language_vectorized_cast.py

```
@@ -0,0 +1,90 @@
+    "float8_e4m3": torch.float8_e4m3fn,
+    "float8_e5m2": torch.float8_e5m2,
+}
+
+
+@tilelang.jit(compile_flags=['-DENABLE_BF16'])
+def vectorized_cast_kernel(M: int, dtype_A: str, dtype_B: str):
+    assert M % 256 == 0
+
+    @T.prim_func
+    def main(
+        A: T.Tensor[(M), dtype_A],  # noqa: F821
```

Ruff (0.14.1)

Unused `noqa` directive (unused: `F821`)

Remove unused `noqa` directive

(RUF100)